mobile/Yote/js/modules/product/productActions.js

```diff
@@ -151,10 +152,12 @@ function receiveUpdateProduct(json) {
 }
 
 export function sendUpdateProduct(data) {
+  console.log(data._id); 
   return dispatch => {
     dispatch(requestUpdateProduct(data))
     return callAPI(`/api/products/${data._id}`, 'PUT', data)
     .then(json => dispatch(receiveUpdateProduct(json)))
+    .then(json => dispatch(fetchList())); 
```

Comment:
@grantfowler should this happen here?  I think in the web we're firing an `.invalidateList()` action from the component, yes?